### ISSUE TITLE
dialog popup overflow/scrollbars

### ISSUE THREAD

**Issue by luke-c-sargent** [GMOD/jbrowse#1246]
- JBrowse version: release 1.15.4, 1.15.3
- Web browser being used: firefox, chrome

when viewing a track's details (via left-click or the right-click menu), the window's contents are too big for the div created, and as such scrollbars are created:

![screen shot 2018-10-29 at 12 03 17 pm](https://user-images.githubusercontent.com/11185084/47684709-0d056b80-db91-11e8-88c3-004be5b31f4c.png)

i have made [some changes in a forked repo's branch](https://github.com/luke-c-sargent/jbrowse/commit/2c3d8b6fc280e1d343365930e16ae69e562d338c) mostly trying to remove explicit sizing in favor of automatic sizing, and enlarging the percentage of the screen the dialog pop-up gets. this yields a nicer-looking (to me) window. I've broken up some of the values with `word-wrap: break-word`, but having scroll within the window's textarea is an option too (it seemed less visually appealing to me, but there could be practical reasons to not do this). 

![screen shot 2018-10-29 at 12 00 49 pm](https://user-images.githubusercontent.com/11185084/47684945-f4498580-db91-11e8-8cbb-a97414f993e5.png)

is this a reasonable way to go about addressing these concerns? if so, other, i am happy to issue a PR, though similar dialog pop-ups should probably be styled the same way. if not, are there better means of correcting the overflow/scrollbar issue?

thanks!

**Comment by cmdcolin** [comment]
I know that there are some CSS selectors that apply word wrap on specific fields in the popup like this

css/dialog.scss
```
/*
 force long sequences in feature and alignment detail dialogs to
 wrap at 45em
*/
.detail .value.seq,
 .detail .value.sequence {
    word-wrap: break-word;
    width: 45em;
}
```

In the interim, I might add your fields like

```
.detail .value.seq,
.detail .value.subject,
.detail .value.query,
 .detail .value.sequence {
    word-wrap: break-word;
    width: 45em;
}
```

I see your proposed changes apply word-wrap on all value fields which seems fairly reasonable and might be something worth considering...

**Comment by rbuels** [comment]
@luke-c-sargent could you submit a pull request with those changes? this will just make it easier for us to test and review it.

one review issue I can already see: could you remove the use of the `!important` tags? those tend to cause a lot of problems with people's custom CSS, etc

**Comment by luke-c-sargent** [comment]
i will reform my approach and submit a pull request when things are more orderly; unsurprisingly, `!important` were causing up (down?)-stream changes that were unintended.

**Comment by cmdcolin** [comment]
My paired_read_tracks branch contains logic that applies the width 45em and break-word when the length of string is over 70 and contains no spaces. The reason for the conditional is because if it is applied for all fields then it puts all fields on a newline which was

**Comment by rbuels** [comment]
closing, this is fixed in #1235
